run_dir/static/js/lanes_ordered.js

```diff
@@ -0,0 +1,286 @@
+            </div>
+            <div class="col-6">
+                <v-lanes-ordered-chart></v-lanes-ordered-chart>
+            </div>
+        </div>
+        `
+});
+
+
+app.component('v-lanes-ordered-item', {
+    props: ['key_array', 'current_key', 'key_level', 'local_data'],
+    computed: {
+        /* local data keys without the key 'value' sorted with highext value first*/
+        local_data_keys() {
+            return Object.keys(this.local_data)
+                .filter(key => key !== 'value')
+                .sort((a, b) => this.local_data[b]['value'] - this.local_data[a]['value']);
+        }
+    },
+    methods: {
+        collapse(key) {
+            /* Change focus to the parent */
+            this.$root.in_focus = this.key_array.slice(0, this.key_level - 1);
+            /* Delete the children from the data tree */
+            this.dumpData(key);
+        },
+        dumpData(key) {
+            let key1 = this.key_array[0];
+            let key2 = this.key_array[1];
+            let key3 = this.key_array[2];
+
+            // A function to delete everything except 'value' from a dictionary
+            function dumpDataHelper(dictionary) {
+                for (let [my_key, value] of Object.entries(dictionary)) {
+                    if (my_key !== 'value') {
+                        delete dictionary[my_key]
+                    }
+                }
+            }
+            switch (this.key_level) {
+                case 1: {
+                    // Delete everything except 'value' from statistics_data
+                    dumpDataHelper(this.$root.statistics_data[key])
+                    break;
+                }
+                case 2: {
+                    dumpDataHelper(this.$root.statistics_data[key1][key])
+                    break;
+                }
+                case 3: {
+                    dumpDataHelper(this.$root.statistics_data[key1][key2][key])
+                    break;
+                }
+            }
+        },
+        fetchData(key) {
+            let args = this.key_array.slice(0, this.key_level - 1);
+            args.push(key);
+            this.$root.fetchStatistics(...args);
+            this.$root.in_focus = args;
+        },
+        focus_on(key) {
+            let args = this.key_array.slice(0, this.key_level);
+            args.push(key);
+            this.$root.in_focus = args;
+        },
+        has_leaves(category) {
+            return (Object.keys(this.local_data[category]).length > 1) && (this.key_level < 5)
+        }
+    },
+    template:
+        /*html*/`
+        <template v-for="category in this.local_data_keys" :key="category">
+            <div :class="has_leaves(category) ? 'ml-4 mb-4' : 'ml-4'">
+                <h3 class="mb-2">
+                    <v-template v-if="key_level == 4">
+                        <a :href="'project/' + category">{{category}}</a> {{this.local_data[category]['value']}}
```

Comment:
Maybe open the project in a new tab?
```suggestion
                        <a :href="'project/' + category" target="_blank">{{category}}</a> {{this.local_data[category]['value']}}
```

Reply:
Oh yes, that's a good suggestion!